ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-cert-manager-1.11.yaml

```diff
@@ -0,0 +1,126 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+  cert-manager-cainjector:
+    name: cert-manager-cainjector
+    namespace: ci
+    tag: v1.11.4
+  cert-manager-controller:
+    name: cert-manager-controller
+    namespace: ci
+    tag: v1.11.4
+  cert-manager-webhook:
+    name: cert-manager-webhook
+    namespace: ci
+    tag: v1.11.4
+  ocp_builder_rhel-9-golang-1.19-openshift-4.13:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.19-openshift-4.13
+binary_build_commands: make build --warn-undefined-variables
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: images/ci/Dockerfile
+  from: base
+  inputs:
+    ocp_builder_rhel-9-golang-1.19-openshift-4.13:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.19-openshift-4.13
+  to: cert-manager-operator
+operator:
+  bundles:
+  - as: cert-manager-operator-bundle
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: openshift.io/cert-manager-operator:latest
+    with: pipeline:cert-manager-operator
+  - pullspec: quay.io/jetstack/cert-manager-controller:v1.11.4
+    with: cert-manager-controller
+  - pullspec: quay.io/jetstack/cert-manager-webhook:v1.11.4
+    with: cert-manager-webhook
+  - pullspec: quay.io/jetstack/cert-manager-cainjector:v1.11.4
+    with: cert-manager-cainjector
+promotion:
+  namespace: ocp
+  name: cert-manager-1.11
+releases:
+  initial:
+    integration:
+      namespace: ocp
+      name: cert-manager-1.11
+  latest:
+    integration:
+      include_built_images: true
+      name: cert-manager-1.11
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: verify
+  commands: |
+    make verify -k --warn-undefined-variables
+  container:
+    from: src
+- as: verify-deps
+  commands: |
+    make verify-deps -k --warn-undefined-variables
+  container:
+    from: src
+- as: unit
+  commands: |
+    make test --warn-undefined-variables
+  container:
+    from: src
+- as: e2e-operator
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OO_INDEX: ci-index-cert-manager-operator-bundle
+    env:
+      OO_CHANNEL: stable-v1
+      OO_INSTALL_NAMESPACE: cert-manager-operator
+      OO_PACKAGE: cert-manager-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: test
+      cli: latest
+      commands: make test-e2e
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: optional-operators-ci-aws
+- as: e2e-operator-upgrade
+  optional: true
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OO_INDEX: ci-index-cert-manager-operator-bundle
+    env:
+      OO_CHANNEL: stable-v1
+      OO_INSTALL_NAMESPACE: cert-manager-operator
+      OO_LATEST_CSV: cert-manager-operator.v1.11.1
```

Comment:
1.11.1 or 1.11.4?

Reply:
Trying to see if rehearsals pass first.
Ideally, would be v1.11.4: yes (once most rehearsal tests pass).

Reply:
Followed up in https://github.com/openshift/release/pull/40954.